### PR TITLE
PCLM Beta release

### PR DESCRIPTION
**Pull request by Nidhi251289**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md [hunk 1]
```diff
@@ -1863,17 +1874,46 @@ Enabling the Development Mode option will allow users to change the type of an e
...
+
+When type is selected as Claim, a list of license will be shown which will show the licenses from license bundle which are claimed in the namespace.
+
+{{< figure src="/attachments/developerportal/deploy/private-cloud/private-cloud-cluster/LicenseClaim.png" >}}
+
+When type is selected as Operator, a list of all the Operator license in the bundle will be shown.
+
+{{< figure src="/attachments/developerportal/deploy/private-cloud/private-cloud-cluster/OperatorList.png" >}}
+
+When type is selected as Runtime, a list of all the Operator license in the bundle will be shown.
```
Comment (katarzyna-koltun-mx) [inline content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: @Nidhi251289 Shouldn't this be "Runtime"?

**Comment by katarzyna-koltun-mx** [comment]
@Nidhi251289 Reviewed, 1 open question above, otherwise can publish when you give the go ahead.

File: content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md (hunk 1 above)
Comment (Nidhi251289) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: indeed, it should be Runtime

Comment (katarzyna-koltun-mx) [reply content/en/docs/developerportal/deploy/private-cloud/private-cloud-cluster/_index.md]: Done.

**Comment by katarzyna-koltun-mx** [comment]
Link errors in this PR seem to be because the actual PCLM file is in another PR (https://github.com/mendix/docs/pull/5581/files).

**Pull request merged**